lnbits/docker-compose.yml

```diff
@@ -32,3 +32,6 @@ services:
       LNBITS_DEFAULT_WALLET_NAME: "LNbits wallet"
       LNBITS_DISABLED_EXTENSIONS: "amilk"
       LNBITS_ADMIN_EXTENSIONS: "ngrok"
+      
+      # Cashu
+      CASHU_PRIVATE_KEY: ${APP_SEED}
```

Comment:
> Cashu mint and wallet
Here you can create multiple cashu mints that you can share. Each mint can service many users but all ecash tokens of a mint are only valid inside that mint and not across different mints. To exchange funds between mints, use Lightning payments.

> Important
If you are the operator of this LNbits instance, make sure to set CASHU_PRIVATE_KEY="randomkey" in your configuration file. Do not create mints before setting the key and do not change the key once set.

I would like the private key to be a value derived from entropy already available in the umbrel application environment in order to prevent complication during installation, operation and restore of a mint. I've seen other apps use the APP_SEED or derived value. Is this acceptable in this scenario?

Reply:
Tagging @callebtc for visibility.

Reply:
This is the correct way to access this 👍 . In this case it will be a 256 bit hex string deterministically derived from 128 bits of cryptographically secure entropy. Is that all that is needed  for `CASHU_PRIVATE_KEY`?

Reply:
According to Cashu documentation it can be an arbitrary string that doesn't have any special requirements as long as it doesn't change for the lifetime of the mint.